lib/dockerize_rails/docker_commands/docker_build.rb

```diff
@@ -80,8 +80,12 @@ def self.build_postgres
       def self.build_docker_image(dockerfile, repo)
         image = Docker::Image.build_from_dir('.', dockerfile: dockerfile) do |output|
           if DRNameSpace.namespace.stream_log
-            if (log = JSON.parse(output)) && log.key?('stream')
-              $stdout.puts log['stream']
+            begin
+              if (log = JSON.parse(output)) && log.key?('stream')
+                $stdout.puts log['stream']
+              end
+            rescue JSON::ParserError => exception
+              puts " ==> JSON Parse Error #{exception.to_s}".red
```

Comment:
__[rubocop]__
* Redundant use of `Object#to_s` in interpolation. [:ref](https://github.com/bbatsov/ruby-style-guide#no-to-s)

<!-- Comment by [SideCI](https://sideci.com) -->